src/site/content/en/blog/samesite-cookie-recipes/index.md

```diff
@@ -0,0 +1,324 @@
+["Handling incompatible clients"](#handling-incompatible-clients) below.
+
+```text
+Set-Cookie: third_party_var=value; SameSite=None; Secure
+```
+
+### Identifying cookie usage
+
+As of Chrome 77, you will be able to see warnings in the console cross-site
+cookies that do not currently have a `SameSite` attribute and cookies that have
+been marked with `SameSite=None` but are missing `Secure`.
+
+<figure class="w-figure  w-figure--center">
+  <img src="chrome-console-warning.png"
+      alt="Chrome console warnings for SameSite cookie misconfiguration"
+      style="max-width: 35vw;">
+  <figcaption class="w-figcaption">
+    Chrome console warnings for <tt>SameSite</tt> cookie misconfiguration.
+  </figcaption>
+</figure>
+
+For missing `SameSite` attributes you will see:
+
+> "A cookie associated with a cross-site resource at (cookie domain) was set
+> without the `SameSite` attribute. A future release of Chrome will only deliver
+> cookies with cross-site requests if they are set with `SameSite=None` and
+> `Secure`. You can review cookies in developer tools under
+> Application>Storage>Cookies and see more details at
+> [https://www.chromestatus.com/feature/5088147346030592](https://www.chromestatus.com/feature/5088147346030592)
+> and
+> [https://www.chromestatus.com/feature/5633521622188032](https://www.chromestatus.com/feature/5633521622188032)."
+
+And for `None` without `Secure`, you will see:
+
+> "A cookie associated with a resource at (cookie domain) was set with
+> `SameSite=None` but without `Secure`. A future release of Chrome will only
+> deliver cookies marked `SameSite=None` if they are also marked `Secure`. You
+> can review cookies in developer tools under Application>Storage>Cookies and
+> see more details at
+> [https://www.chromestatus.com/feature/5633521622188032](https://www.chromestatus.com/feature/5633521622188032)."
+
+Each of these warnings will contain the cookie domain. If you're responsible for
+that domain, then you will need to update the cookies. Otherwise, you may need
+to contact the owner of the site or service responsible for that cookie to
+ensure they making the necessary changes. The warnings themselves do not affect
+the functionality of the site, this is purely to inform developers of the
+upcoming changes.
+
+### Handling incompatible clients
+
+As these changes to include `None` and update default behavior are still
+relatively new, there are inconsistencies amongst browsers as to how these
+changes are handled. You can refer to the
+[updates page on chromium.org](https://www.chromium.org/updates/same-site/incompatible-clients)
+for the issues currently known, however it's not possible to say if this is
+exhaustive. While this is not ideal, there are workarounds you can employ during
+this transitionary phase. The general rule though is to treat incompatible
+clients as the special case - do not create an exception for browsers
```

Comment:
'Do not create...' is the start of a new sentence.

Reply:
Split.